packages/elasticsearch-asset-apis/test/api-with-dataFrams-spec.ts

```diff
@@ -0,0 +1,484 @@
+import 'jest-extended';
```

Comment:
The name of this file has spelling mistakes, it should also be more clear, `reader-api-with-data-frames-spec.ts`